ribs/emitters/_gradient_arborescence_emitter.py

```diff
@@ -32,7 +32,7 @@ class GradientArborescenceEmitter(EmitterBase):
             :class:`ribs.archives.GridArchive`.
         x0 (np.ndarray): Initial solution.
         sigma0 (float): Initial step size / standard deviation.
-        step_size (float): Step size for the gradient optimizer
+        step_size (float): Step size for the gradient optimizer.
```

Comment:
Do we want this to be a kwargs to the optimizers?

Reply:
All optimizers use it and it is a commonly set parameter, so let's keep it